conda/build_pytorch.sh

```diff
@@ -219,7 +220,8 @@ elif [[ "$OSTYPE" == "msys" ]]; then
     pushd $tmp_conda
     export PATH="$(pwd):$(pwd)/Library/usr/bin:$(pwd)/Library/bin:$(pwd)/Scripts:$(pwd)/bin:$PATH"
     popd
-    retry conda install -yq conda-build
+    # TODO(huydhn): We can revert the pin after https://github.com/conda/conda-build/issues/5167 is resolved
```

Comment:
Since failure is with MacOS only we probably don't need to apply this constraint to windows. Only MacOS should be enought

Reply:
I hope that conda doesn't put us into a situation where both pinning an older version and using the latest version fail :)